src/qml/FeatureListForm.qml

```diff
@@ -15,12 +15,15 @@
  *                                                                         *
  ***************************************************************************/
 
-import QtQuick 2.3
-import QtQuick.Layouts 1.1
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
 import QtQuick.Dialogs 1.2
+
 import org.qgis 1.0
 import org.qfield 1.0
 import Theme 1.0
+import "."
```

Comment:
What's that one for?

Reply:
Button { id: deleteButton... }

Reply:
Next step: kill all those "." imports and put those into a library like Theme

Reply:
@m-kuhn , I've actually tried, and couldn't get it to compile then. Might have just been me juggling with too many PRs.